lib/license_finder/cli/main.rb

```diff
@@ -91,7 +91,13 @@ def self.shared_options
       shared_options
       def project_roots
         config.strict_matching = true
-        aggregate_paths
+        paths = aggregate_paths
+        filtered_project_roots = Scanner.remove_subprojects(paths)
+        if filtered_project_roots.empty?
+          say(paths)
```

Comment:
So, we are saying that if every path is a subproject, 'project_roots' will still print out all paths, including subprojects?  Isn't that a bit confusing especially if there is at least one project that is a root project, all subprojects will be filtered out in the output.